### DISCUSSION TITLE
Where should HTML syntax go? What should the header be?

### DISCUSSION THREAD

**Discussion by estelle** [mdn/mdn-community#525] [Content] [answered]
Every HTML element has a table with syntax with the following information:

* Content categories
* Permitted content 
* Tag omission 
* Permitted parents 
* Implicit ARIA role 
* Permitted ARIA roles 	
* DOM interface 	

This appears in different locations on different pages, and with different headers. For example, [`<dialog>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) and 99 other others hav it right after the summary paragraph with no header, before ### attributes.  On [`<video>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#technical_summary) and 13 other elements display it just before the specifications header  under "Technical Summary", and 45 or so that do it some other way, like [`<a>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) that calls it properties. 

I believe the correct way is #technical summary before specifications. Want to confirm before updating all.

**Comment by hamishwillee** [comment, marked as answer]
@estelle Does this answer your question?: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Page_types/HTML_element_page_template

If not, on completion of this discussion that doc should be updated to match.

**Reply by estelle** [reply]
Thank you. I did see that. I just wanted to make sure that is current before moving forward.

**Reply by estelle** [reply]
If an element is deprecated, like acronym, should the section
1)  be left out
2) have a title, but read "not applicable" since it shouldn't be used.
3) included

An example would be [`<acronym>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/acronym) does not include it while [`<applet>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/applet) does.

**Reply by estelle** [reply]
https://github.com/mdn/content/pull/22831/files

**Reply by hamishwillee** [reply]
Hi @estelle 
I couldn't tell you for sure if it is up to date - sorry.

My take on this is that there should be no difference between a deprecated and non-deprecated document other than the page front-metadata key "deprecated" and the deprecated macro.

So if this is present in the page structure it should be present in acronym.

That said, I would not waste my time back adding anything to a deprecated topic unless it was easy to do.

Does that make sense?
